cmd/mc-router/main.go

```diff
@@ -40,6 +40,7 @@ type Config struct {
 	InKubeCluster        bool     `usage:"Use in-cluster kubernetes config"`
 	KubeConfig           string   `usage:"The path to a kubernetes configuration file"`
 	MetricsBackend       string   `default:"discard" usage:"Backend to use for metrics exposure/publishing: discard,expvar,influxdb"`
+	SendProxyProtocol    bool     `default:"false" usage:"Send PROXY protocol to backend servers"`
```

Comment:
Nitpick: I would prefer `UseProxyProtocol`

Reply:
BTW the pickiness here is only because this is the "public interface" of the app

Reply:
I thought SendProxyProtocol would be better as I was thinking to add recieve as well

Reply:
solved 👍🏼

Reply:
> I thought SendProxyProtocol would be better as I was thinking to add recieve as well

That logic makes sense. In any case, can still use "receive" or "accept" terminology if/when needed.